app/Controllers/FrontendController.php

```
@@ -971,9 +971,21 @@ private function buildWhereConditions(array $filters, mysqli $db): array
         }
 
         if (!empty($filters['editore'])) {
-            $conditions[] = "e.nome = ?";
-            $params[] = $filters['editore'];
-            $types .= 's';
+            // Match the publisher whether it is the book's primary (the joined
+            // `e`) or a secondary one in the multi-publisher junction (issue
+            // #143), so the catalog filter results agree with the publisher
+            // facet count (which already counts secondaries). Gate the junction
+            // subquery on table existence (pre-migration safety).
+            if (\App\Support\SchemaInfo::hasLibriEditori($db)) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SchemaInfo' in method 'buildWhereConditions'. (undefined)

(StaticAccess)